phpBB/phpbb/files/types/type_interface.php

```diff
@@ -21,7 +21,7 @@ interface type_interface
 	 * Handle upload for upload types. Arguments passed to this method will be
 	 * handled by the upload type classes themselves.
 	 *
-	 * @return \phpbb\files\filespec|bool Filespec instance if upload is
+	 * @return \phpbb\files\filespec_storage|\phpbb\files\filespec|bool Filespec instance if upload is
```

Comment:
filespec probably not, that class is not used anymoer

Reply:
Since it still exists and is at least used in that way in the code, this has to be defined this way. Can be removed when we remove filespec.